jaxdecomp/fft.py

```diff
@@ -1,9 +1,11 @@
 import jax.numpy as jnp
 from jax._src.numpy.fft import _fft_norm
```

Comment:
`_fft_norm` is redefined below (code copied from jax source)

Reply:
right, can we remove the code below and just use this function?